### PR TITLE
Borrowing list detail

### PR DESCRIPTION
**Pull request by Andriy-Makarenko**

### CODE REVIEW

File: borrowing/serializers.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "book",
+            "author",
+            "user",
+            "borrow_date",
+            "expected_return_date"
+        )
+
+
+class BorrowingDetailSerializer(BorrowingListSerializer):
+    cover = serializers.CharField(source="book.cover", read_only=True)
```
Comment (LiudmylaKulinchenko) [inline borrowing/serializers.py]: use BookSerializer instead

File: borrowing/views.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+from borrowing.models import Borrowing
+from borrowing.serializers import (
+    BorrowingSerializer,
+    BorrowingDetailSerializer,
+    BorrowingListSerializer
+)
+
+
+class BorrowingViewSet(viewsets.ModelViewSet):
```
Comment (LiudmylaKulinchenko) [inline borrowing/views.py]: Don't use ViewSet, should implemet only GET and POST

File: borrowing/urls.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from rest_framework import routers
+
+from borrowing.views import BorrowingViewSet
+
+router = routers.DefaultRouter()
```
Comment (LiudmylaKulinchenko) [inline borrowing/urls.py]: remove router

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]

**Review by shurushku — APPROVED** [review]

**Comment by Andriy-Makarenko** [comment]
Ура

**Pull request merged**